app/src/components/UI/Molecules/YourAccount/YourAccountOption.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from "react";
+import { Container, Icon, Header } from "semantic-ui-react";
+import styles from "./youracctbox.module.css";
+
+const YourAccountOption: FC<{ header: string; iconName: string }> = ({
+  header,
+  iconName,
+}) => {
+  return (
+    <Container className={styles.wrapper}>
+      <Header as={"h1"} className={styles.header}>
+        {header}
+      </Header>
+      <Container className={styles.iconWrap}>
+        {iconName! === "favorite" ? (
```

Comment:
You don't need the `!` because `iconName` is never null.

Reply:
Ok.